Assets/Scripts/Actors/ActorComponents/PlayerActorPhysics.cs

```diff
@@ -1,135 +0,0 @@
-﻿using UnityEngine;
-using System.Collections;
-
-[RequireComponent( typeof( PlayerActor ) )]
-public class PlayerActorPhysics : ActorPhysics
-{
-	PlayerActor _actor;
-	bool _jumpButtonDown = false;
-	bool _isGrabbing = false;
-
-	public override void Awake()
-	{
-		base.Awake();
-
-		_actor = GetComponent<PlayerActor>();
-		ChangeState( ActorStates.Grounded );
-	}
-
-	void Update()
-	{
-		if ( Input.GetKeyDown( KeyCode.Escape ) )
```

Comment:
This could be split into it's own component or combined with the reset scene component. It needs to be somewhere or else a release build won't close when you hit esc.